custom_components/localtuya/strings.json

```diff
@@ -39,5 +36,98 @@
             }
         }
     },
+    "options": {
+        "step": {
+            "init": {
+                "title": "LocalTuya Configuration",
+                "description": "Please select the desired actionSSSS.",
+                "data": {
+                    "add_device": "Add a new device",
+                    "edit_device": "Edit a device",
+                    "delete_device": "Delete a device",
+                    "setup_cloud": "Reconfigure Cloud API account"
```

Comment:
Can't find the exact right spot; but on this UI - help link just points to the github repo.
![image](https://user-images.githubusercontent.com/365751/171171254-e85b8a64-0f78-4fb6-9076-7161c82370d6.png)

Might be better to point to the README or a specific relevant subsection (not sure how thought)

Reply:
OK I think I found it, it's the "documentation" field in the manifest.json . It's not something you can make window-specific (I mean, you only have one link, you can't have a different link to different subsection for every configuration step, which is what you are proposing, probably). So I'd keep it like this: it points to the repo, which is what is commonly done in all custom components.